eth/gasprice/gasprice.go

```diff
@@ -160,6 +177,14 @@ func (gpo *Oracle) SuggestPrice(ctx context.Context) (*big.Int, error) {
 	return price, nil
 }
 
+func (gpo *Oracle) minPrice(num *big.Int) *big.Int {
+	if gpo.defaultPrice != nil {
+		return gpo.defaultPrice
+	}
+	const blockOffset = 12 // look ~1 minute ahead, since we are suggesting gas for near-future txs
+	return DefaultFn(gpo.backend.ChainConfig())(new(big.Int).Add(big.NewInt(blockOffset), num))
```

Comment:
Is one minute long enough here?

Reply:
Maybe do a bit longer just in case. An hour?